packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -548,8 +660,9 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
   double get _afterExtent => _isReversed ? _lastMetrics!.extentBefore : _lastMetrics!.extentAfter;
   // Padding of the thumb track.
   double get _mainAxisPadding => _isVertical ? padding.vertical : padding.horizontal;
+  double get _buttonSize => showButtons ? math.min(buttonsShape!.getButtonSize(), math.max((_lastMetrics!.viewportDimension - 2 * mainAxisMargin - _mainAxisPadding) / 2.0, 0.0)) : 0.0;
```

Comment:
This is something users may want control over. Might be worth exposing in theming, but we could wait and see if it is something folks actually want first.